db/migrate/20200708051009_cap_bookmark_name_at_100_characters.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CapBookmarkNameAt100Characters < ActiveRecord::Migration[6.0]
+  def up
+    DB.exec("UPDATE bookmarks SET name = LEFT(name, 100)")
```

Comment:
minor ... no point updating name when it did not change .... `WHERE name IS NOT NULL name <> LEFT(name, 100)`

Reply:
:+1: